src/TraceEvent/TraceEvent.csproj

```diff
@@ -71,6 +71,7 @@
 
   <!-- *** SourceLink Support *** -->
   <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
+  <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
```

Comment:
❗️ I believe this new dependency needs one (or more if it has transitive dependencies) new blocks like this so the distribution is still complete:

https://github.com/microsoft/perfview/blob/b2f8e0ebd0434fac7e40960dc5521ce121c26a72/src/PerfView/PerfView.csproj#L250-L256

Reply:
Done.